app/models/course.rb

```diff
@@ -66,6 +67,12 @@ class Course < ApplicationRecord
     other: "O",
   }
 
+  enum level: {
+    primary: 'Primary',
+    secondary: 'Secondary',
+    further_education: 'Further education'
+  }, _prefix: :course
```

Comment:
Shoukld this be suffix instead of prefix?

Reply:
Good spot